src/apps/sctp_test.c

```diff
@@ -612,7 +612,17 @@ int socket_r(void)
 		fprintf(stderr, "SCTP_EVENTS: error: %d\n", error);
 		exit(1);
 	}
-
+        if (max_stream > 0) {
+        	struct sctp_initmsg initmsg;
+        	memset(&initmsg, 0, sizeof(struct sctp_initmsg));
+        	initmsg.sinit_num_ostreams = max_stream;
+                int i 
+                error = setsockopt(sk, IPPROTO_SCTP, SCTP_INITMSG, &initmsg, sizeof(struct sctp_initmsg));
```

Comment:
And adjust the indentation here. There is an extra whitespace and the line is too long.